server/controllers/profile.js

```diff
@@ -41,3 +41,18 @@ exports.loadProfile = asyncHandler(async (req, res, next) => {
     },
   });
 });
+
+// @route GET /profile/listing
+// @desc Get sitter profile data
+// @access Private
+exports.loadSitters = asyncHandler(async (req, res, next) => {
+  const profile = await Profile.find({ sitter: true });
```

Comment:
If no profiles are found, an empty array will be returned which can be sent as a response to the FE (where we can conditionally display a message)

Reply:
I have added the conditions on FE instead of BE. Please let me know if it is okay to do this way